pages/docs/how-to-guides/events.mdx

```diff
@@ -45,7 +45,9 @@ _If you need help on how to edit this file, please see the <Link href="/docs/how
     "road": "Messe Berlin South Entrance & CityCube",
     "city": "Jafféstrasse",
     "state": "Berlin",
-    "country": "Germany"
+    "country": "Germany",
+    "lat":37.774929,
+    "lon":-122.419418
```

Comment:
We can't ask people to add the lat/lon this should be automated, we have functionality to do this for the user location, I think you can extract that and make it reusable

Reply:
Automated fetching of latitude and longitude by event city. Passed city in the url and got latitude and longitude

Reply:
This does not to be in the documentation because it is not provided by the user.

Reply:
Docs corrected